rate: 10
filtering_method: "median"
bounding_box_percet_reduction: 0.6
max_traslation_distance: 2.0
filter_radious: 0.8
search_time: 5.0
front_image_cov1: 1.0
front_image_cov2: 1.0